wiki/general/tutorials/getting-started-.md

```diff
@@ -0,0 +1,120 @@
+In that tutorial we will cover how to use Launchpad to generate a WordPress plugin.
+
+### Create the project
+
+#### Generate the project using composer
+The first step in creating the project itself is by using the composer command line with the following input: `composer create-project wp-launchpad/launchpad my-plugin`
+
+Once this is completed, you should be able to see a new folder with the code for your project inside it. However, it is not yet ready to be used as it is still Launchpad default code and needs to be initialized.
+
+#### Initialize project with launchpad command
+
+As multiple projects can exist on the same WordPress website at once, it is important to containerize all of them, and for that purpose we will use a specific namespace for our code.
+
+To do so with Launchpad, we need to run the following command in the terminal:
+`bin/generator initialize`
+
+Once the command has been executed, it should ask you for your plugin name, and it will generate the necessary changes from it.
+
+You can assert the command executed successfully by checking if the name from your plugin file matches what was the name from your plugin.
+
+### Install a module in your project using composer
+
+Now that the project is set up properly, it is now time to install some modules to gain time and effort when working on your project.
+
+A good option would be to use the WordPress options facade to inside the module `wp-launchpad/framework-options`.
+
+For that we will need to install the corresponding take off package `wp-launchpad/framework-options-take-off` that will set it up automatically for us.
+
+For that we will have to run the following command: `composer install wp-launchpad/framework-options-take-off`
+
+Once this is done, then you can assert everything went as expected by looking that the provider has been added to `configs/providers.php` and that we have the matching package installed in the `composer.json` file.
+
+It is possible to find a list of every module available on [the module list page](../../modules/listing.md).
+
+### Create a new subscriber
+
+Once the project is now set up properly.
+It is now a great time to create our first subscriber.
+
+
+#### Create a new subscriber
+
+To create a subscriber, we can create any class with the name of your choice.
+
+What will make it become a subscriber is actually adding a docblock with the annotation `@hook`.
+
+This annotation signals that this method should be used as an event handler for some event.
```

Comment:
Is there a more detailed documentation about this syntax? If yes, then link it. Otherwise, it will be needed: for someone coming from "normal" WP, this is a big change and it's not clear how this replaces add_action for instance.